pkg/runs/timeUnit.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package runs
+
+const (
+	TIME_UNIT_WEEKS   = "weeks"
```

Comment:
I would expect to see TIME_UNIT_WEEKS_LONG and TIME_UNIT_WEEKS_SHORT... then we can use a symbolic for 'w' also...